model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/updater/liquibase/MapJpaLiquibaseUpdaterProvider.java

```diff
@@ -163,7 +164,7 @@ protected Status validateChangeSet(Liquibase liquibase, String changelog) throws
     }
 
     private Liquibase getLiquibase(Class modelType, Connection connection, String defaultSchema) throws LiquibaseException {
-        LiquibaseConnectionProvider liquibaseProvider = session.getProvider(LiquibaseConnectionProvider.class);
+        LiquibaseConnectionProvider liquibaseProvider = session.getProvider(LiquibaseConnectionProvider.class, MapLiquibaseConnectionProviderFactory.PROVIDER_ID);
         String changelog = "META-INF/jpa-" + ModelEntityUtil.getModelName(modelType) + "-changelog.xml";
         if (changelog == null) {
```

Comment:
Just spotted this: `changelog` is never `null` although `ModelEntityUtil.getModelName(modelType)` can be.
cc: @vramik

Reply:
Added a fix for this as part of this PR.